src/tests/ftest/control/ms_resilience.py

```diff
@@ -66,6 +66,22 @@ def create_pool(self):
             time.sleep(1)
         self.log.info("Found pool in system.")
 
+    def get_hostname(self, host_addr):
```

Comment:
(pylint-no-self-use) Method could be a function

Reply:
your mom could be a function